oxford-metadata.ttl

```diff
@@ -168,15 +168,15 @@
     rdfs:label "Pumps" ;
     rdfs:comment "Terms related to proteins that use energy in ATP to pump ions against their electrochemical gradients." .
 
-:MembraneExchanger
+:MembraneExchangerRelated
     a :CellMembraneCurrentRelated ;
     rdfs:label "Exchangers" ;
-    rdfs:comment "Proteins that use the energy of one ionic species moving down an electrochemical gradient, to move another species against its gradient." .
+    rdfs:comment "Terms related to proteins that use the energy of one ionic species moving down an electrochemical gradient, to move another species against its gradient." .
 
 :MembraneIonChannel
```

Comment:
You've not renamed this one, for instance. And other terms down to SRIonChannel. But possibly we don't need to since their existing names don't imply they're currents in the same way? What do you think @mirams ?

Reply:
Yeah, for consistency I think these probably want to become 'Related' suffix, because they are labelling things that aren't actually the channel/current but properties of it. Same for MembranePotassiumChannel etc.

Reply:
ok @mirams @MichaelClerx @jonc125 Just ypdated it please have a look if this is what you meant